rpc/src/commands/get_blocks_in_chain.rs

```diff
@@ -1,15 +1,36 @@
 use super::RPCClientCommand;
+use crate::responses::BlocksInChainResponse;
+
+enum ChainType {
```

Comment:
This should be higher up in the crate, at the rpc command level probably. Also, should probably add Test as one of the options

Reply:
agreed, good catch

Reply:
actually now that i think about it, it might even be worth it to put it in it's own module seeing as we will almost surely eventually want to move structs/enums as common as `ChainType` into a different `lib` (outside of `rpc`). 

Having to force the import now means less work later

Reply:
I think `ChainType` will probably be set at the RPCClient instantiation level, and maybe not used anywhere else. I agree that moving to different `lib` is probably the move for now